src/agents/policy_retriever_agent.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+sys.path.append("..")
+
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_community.vectorstores import Chroma
+from openai import OpenAI, AzureOpenAI
+
+import config as cfg
+from helpers.embedding_helpers import OpenAIEmbeddingFunction
+
+# Environment setup
+os.environ['KMP_DUPLICATE_LIB_OK'] = 'True'
+
+# Fuse to blob storage to access chroma db and policy documents
+os.system('sudo blobfuse /home/azureuser/cloudfiles/code/blobfuse/rndaistoragemeldingen --tmp-path=/mnt/resource/blobfusetmp_meldingen --config-file=/home/azureuser/cloudfiles/code/blobfuse/fuse_connection_meldingen.cfg -o attr_timeout=3600 -o entry_timeout=3600 -o negative_timeout=3600 -o allow_other -o nonempty')
+
+class PolicyRetrieverAgent:
```

Comment:
Naming of the whole class (and containing folder btw) -> these are all **_tools_**, not separate **_agents_**. I can find some articles around that properly explain the difference between tools and agents if that doesn't easily make sense :)

Reply:
I'll change this naming